backend/src/Designer/Services/Implementation/TextResourceService.cs

```diff
@@ -50,11 +47,12 @@ public async Task UpdateTextResourcesAsync(string org, string app, string shortC
             List<FileSystemObject> folder = await _giteaApiWrapper.GetDirectoryAsync(org, app, textResourcesPath, shortCommitId);
             if (folder != null)
             {
-                folder.ForEach(async textResourceFromRepo =>
+                // TODO: Add Cancellation token to signature of the method and to the methods used in loop and convert loot to Parallel.ForEachAsync
```

Comment:
should we create a issue within Github for this and add the issue number to the comment? 😊

Reply:
added